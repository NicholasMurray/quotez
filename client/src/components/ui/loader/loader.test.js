import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import Loader from '.';

test('it renders the loader correctly', () => {
  const tree = TestRenderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
