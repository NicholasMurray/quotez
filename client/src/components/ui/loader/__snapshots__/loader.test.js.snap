// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the loader correctly 1`] = `
.c0 {
  height: 100px;
  width: 100px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 svg {
  fill: #66c7f4;
}

<div
  className="c0"
>
  <svg
    enableBackground="new 0 0 0 0"
    id="L5"
    version="1.1"
    viewBox="0 0 100 100"
    x="0px"
    xmlSpace="preserve"
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
    y="0px"
  >
    <circle
      cx="6"
      cy="50"
      r="6"
      stroke="none"
    >
      <animateTransform
        attributeName="transform"
        begin="0.1"
        dur="0.5s"
        repeatCount="indefinite"
        type="translate"
        values="0 15 ; 0 -15; 0 15"
      />
    </circle>
    <circle
      cx="30"
      cy="50"
      r="6"
      stroke="none"
    >
      <animateTransform
        attributeName="transform"
        begin="0.2"
        dur="0.5s"
        repeatCount="indefinite"
        type="translate"
        values="0 10 ; 0 -10; 0 10"
      />
    </circle>
    <circle
      cx="54"
      cy="50"
      r="6"
      stroke="none"
    >
      <animateTransform
        attributeName="transform"
        begin="0.3"
        dur="0.5s"
        repeatCount="indefinite"
        type="translate"
        values="0 5 ; 0 -5; 0 5"
      />
    </circle>
  </svg>
</div>
`;
